tracking/imu/lsm6ds3trc.h

```diff
@@ -0,0 +1,22 @@
+#ifndef IMU_LSM6DS3TRC_H
+#define IMU_LSM6DS3TRC_H
+
+#include "../../lib/lsm6ds3tr-api/lsm6ds3tr-c_reg.h"
+
+#include <furi_hal.h>
+
+#include "imu.h"
+
+// External variable declaration
+extern stmdev_ctx_t lsm6ds3trc_ctx;
```

Comment:
No need to define it here (doesn't matter though, as it's better to delete the file anyway).